Scripts/Python/kdshGlowInTheDark.py

```diff
@@ -538,3 +513,11 @@ def OnTimer(self,id):
         if id == 5:
             PtDebugPrint("kdshGlowInTheDark: It's been one second. FF'ing floor to glow state.")
             respFloorGlow.run(self.key,fastforward=1)
+
+        if id == 6:
+            note = ptNotify(self.key)
+            note.clearReceivers()
+            note.addReceiver(self.key)
+            note.setActivate(1)
+            note.addVarNumber("foo",id)
+            note.send()
```

Comment:
What's this for?

Reply:
It triggers the early release of the top exclude region.